bower.json

```diff
@@ -22,6 +22,7 @@
     "angular-nvd3": "^1.0.7",
     "string.startsWith": "*",
     "angularLocalStorage": "^0.3.2",
-    "ng-walkthrough": "^1.0.0"
+    "ng-walkthrough": "^1.0.0",
+    "animate.css": "^3.5.2"
```

Comment:
Does this restore @krisma's code to animate the start and end points of the trip?